[2022-05-30 20:21:26,890] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [queued]>
[2022-05-30 20:21:27,755] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [queued]>
[2022-05-30 20:21:27,783] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 20:21:27,787] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 20:21:27,792] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 20:21:29,546] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-10-02 06:00:00+00:00
[2022-05-30 20:21:29,679] {standard_task_runner.py:52} INFO - Started process 1290 to run task
[2022-05-30 20:21:29,843] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-10-02T06:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmp_96jm193', '--error-file', '/tmp/tmp1hlwaa46']
[2022-05-30 20:21:29,996] {standard_task_runner.py:80} INFO - Job 136: Subtask ingest
[2022-05-30 20:21:32,338] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [running]> on host e083cbf9476b
[2022-05-30 20:21:33,432] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 20:21:36,056] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-02T06:00:00+00:00
[2022-05-30 20:21:36,099] {logging_mixin.py:115} INFO - yellow_taxi_2021_10 /opt/***/output_2021-10.parquet 2021-10-02T06:00:00+00:00
[2022-05-30 20:21:36,136] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ingest_script.py", line 12, in ingest_callable
    engine = create_engine(f'postgresql://{user}:{password}@{host}:{port}/{db}')
  File "<string>", line 2, in create_engine
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/deprecations.py", line 298, in warned
    return fn(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/create.py", line 518, in create_engine
    u = _url.make_url(url)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/url.py", line 711, in make_url
    return _parse_rfc1738_args(name_or_url)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/url.py", line 772, in _parse_rfc1738_args
    components["port"] = int(components["port"])
ValueError: invalid literal for int() with base 10: 'None'
[2022-05-30 20:21:36,355] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20211002T060000, start_date=20220530T202126, end_date=20220530T202136
[2022-05-30 20:21:36,620] {standard_task_runner.py:97} ERROR - Failed to execute job 136 for task ingest (invalid literal for int() with base 10: 'None'; 1290)
[2022-05-30 20:21:37,288] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-05-30 20:21:37,759] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-30 22:17:50,139] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [queued]>
[2022-05-30 22:17:50,175] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [queued]>
[2022-05-30 22:17:50,177] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:17:50,180] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 22:17:50,182] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:17:50,241] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-10-02 06:00:00+00:00
[2022-05-30 22:17:50,331] {standard_task_runner.py:52} INFO - Started process 1442 to run task
[2022-05-30 22:17:50,418] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-10-02T06:00:00+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmp4q4hgyjo', '--error-file', '/tmp/tmp6igojto3']
[2022-05-30 22:17:50,434] {standard_task_runner.py:80} INFO - Job 224: Subtask ingest
[2022-05-30 22:17:50,726] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [running]> on host 8ac972de2f4f
[2022-05-30 22:17:51,056] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 22:17:51,621] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-02T06:00:00+00:00
[2022-05-30 22:17:51,664] {logging_mixin.py:115} INFO - yellow_taxi_2021_10 /opt/***/output_2021-10.parquet 2021-10-02T06:00:00+00:00
[2022-05-30 22:17:51,741] {logging_mixin.py:115} INFO - connection established successfully, inserting data...
[2022-05-30 22:18:06,564] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2022-05-30 22:18:06,726] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20211002T060000, start_date=20220530T221750, end_date=20220530T221806
[2022-05-30 22:18:06,837] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-30 22:22:17,412] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [queued]>
[2022-05-30 22:22:17,603] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [queued]>
[2022-05-30 22:22:17,606] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:22:17,610] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 22:22:17,613] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:22:18,560] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-10-02 06:00:00+00:00
[2022-05-30 22:22:19,995] {standard_task_runner.py:52} INFO - Started process 1849 to run task
[2022-05-30 22:22:20,102] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-10-02T06:00:00+00:00', '--job-id', '256', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpyh5te46y', '--error-file', '/tmp/tmpj5uepnii']
[2022-05-30 22:22:21,633] {standard_task_runner.py:80} INFO - Job 256: Subtask ingest
[2022-05-30 22:22:28,848] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-10-02T06:00:00+00:00 [running]> on host 8ac972de2f4f
[2022-05-30 22:22:36,265] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 22:22:39,561] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-02T06:00:00+00:00
[2022-05-30 22:22:39,726] {logging_mixin.py:115} INFO - yellow_taxi_2021_10 /opt/***/output_2021-10.parquet 2021-10-02T06:00:00+00:00
[2022-05-30 22:22:40,011] {logging_mixin.py:115} INFO - connection established successfully, inserting data...
[2022-05-30 22:23:46,959] {local_task_job.py:221} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2022-05-30 22:23:49,384] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 1849. PIDs of all processes in the group: [1849]
[2022-05-30 22:23:49,544] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 1849
[2022-05-30 22:24:44,594] {process_utils.py:75} INFO - Process psutil.Process(pid=1849, status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='22:22:19') (1849) terminated with exit code Negsignal.SIGKILL
[2022-05-30 22:24:48,787] {standard_task_runner.py:162} ERROR - Job 256 was killed before it finished (likely due to running out of memory)
